src/index.js

```diff
@@ -1,11 +1,36 @@
 'use strict'
 
 const multiaddr = require('multiaddr')
-const loadCommands = require('./load-commands')
 const getConfig = require('./default-config')
 const getRequestAPI = require('./request-api')
 
-function IpfsAPI (hostOrMultiaddr, port, opts) {
+/**
+ * Create a Api
+ *
+ * @constructor Api
+ * @param {string} hostOrMultiaddr
+ * @param {number} port
+ * @param {object} opts
+ * @returns {Api}
+ *
+ * @example
+ * const ipfsAPI = require('ipfs-api')
+ *
+ * // connect to ipfs daemon API server
+ * const ipfs = ipfsAPI('localhost', '5001', {protocol: 'http'})
+ * // leaving out the arguments will default to these values
+ *
+ * // or connect with multiaddr
+ * const ipfs = ipfsAPI('/ip4/127.0.0.1/tcp/5001')
+ *
+ * // or using options
+ * const ipfs = ipfsAPI({host: 'localhost', port: '5001', procotol: 'http'})
+ */
+function Api (hostOrMultiaddr, port, opts) {
```

Comment:
Why this changed to just `Api` from `IpfsAPI`? Having ipfs in the name helps when dealing with stacktraces